Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Terrain-3 Bugfixes & Changes

Closes #662. Resolves #614 #623 #625 #626 #1056 #1067.

_Given the scope of `terrain-2` has expanded to greater dimensions, its original purpose was to rectify several bugs and inaccuracies. While there is still more work pertaining to issue #342 to be addressed, this PR will cover immediate bugs._


## Changelog:
- Complete rework of the _Venomweb_ Vale region in Tirisfal Glades.
- Added trees to Northern Eastern Kingdoms regions.
- Added "urban" terrain to the areas associated with Capital City (Lordaeron), Stromgrade (Arathi Basin), and Stratholme (Eastern Lordaeron).
- Numerous bugfixes covered here: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/662#issuecomment-977424839

### 1067
- Added a crossing point between province 16 (STONECAIRN) and province 2384 (TREGATE).


## Known Issues:
- Redoing the trees in the Eastern Kingdoms has consequently reset all of the tree placements on the map.

## Planned Changes & Requests
- For the next PR, the rest of the changes from #662 up to the TRANSLATE project will be implemented. Notably including the Silithus overhaul.
- Correction of the remaining inaccuracies listed in #342.


## Tests:

# How to test:
(This section shamelessly copied from @ercarp)

Consider the following when giving your feedback:

**Gameplay:** For pertinent bugs, verify if the bugfixes are working as intended.

**Canonicity:** Compare the map changes to ensure accuracy according to the information made available by official canon.

**Familiarity:** Observe whether or not the changes are recognizable and fitting for their respective locations.

**Aesthetics:** Comment on the general look and appeal on the changes. Suggestions should be as specific as possible.

Additionally, ensure that necessary gameplay features like Holdings are not obscured by map changes such as trees, water, or steep cliffs.

### -- @ValianBlue
